src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/domain/Vedtaksperiode.kt

```diff
@@ -5,11 +5,15 @@ import no.nav.tilleggsstonader.kontrakter.felles.Periode
 import no.nav.tilleggsstonader.kontrakter.periode.avkortFraOgMed
 import no.nav.tilleggsstonader.sak.vedtak.domain.GeneriskVedtak
 import no.nav.tilleggsstonader.sak.vedtak.domain.InnvilgelseEllerOpphørLæremidler
+import no.nav.tilleggsstonader.sak.vedtak.læremidler.dto.VedtaksperiodeStatus
 import java.time.LocalDate
+import java.util.UUID
 
 data class Vedtaksperiode(
+    val id: UUID,
```

Comment:
Hvis man legger inn default initiering her så må man ikke sende med `id = UUID.randomUUID` i alle tester
```suggestion
    val id: UUID = UUID.randomUUID(),
```

Reply:
Jeg tenkte på det, men valgte å droppe det, fordi det da vil skape et inntrykk av at det er backend som genererer disse ID-ene. 

Men i løsningen sendes de jo inn fra frontend.

Reply:
Høres fint ut 👍